src/commands/uplink/GetCurrentMul.ts

```diff
@@ -38,138 +36,29 @@ class GetCurrentMul extends Command {
-
-        // eslint-disable-next-line @typescript-eslint/prefer-for-of
-        for ( let index = 0; index < channels.length; index++ ) {
-            ({value, position} = this.getChannelValue(data, position));
-
-            const channelIndex = channels[index];
-            const channel: IChannel = {value, index: channelIndex};
-
-            map.set(channelIndex, channel);
-        }
-
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return map;
-    }
-
-    protected static setChannels ( channels: Map<number, number> ) : Array<number> {
-        if ( channels.size === 0 ) {
-            throw new Error(`${this.getName()}. Channel map is empty, nothing to do.`);
-        }
-
-        const lastChannel = Array.from(channels.keys()).pop();
-
-        if ( !lastChannel ) {
-            // just to solve "lastChannel is possibly 'undefined'." error
-            throw new Error(`${this.getName()}. Channel map is empty, nothing to do.`);
-        }
-
-        const size = Math.ceil(lastChannel / LAST_BIT_INDEX) + (lastChannel % LAST_BIT_INDEX) ? 1 : 0;
-        const data = Array(size).fill(EXTENDED_BIT);
-
-        channels.forEach((_, channelIndex) => {
-            data[Math.floor(channelIndex / LAST_BIT_INDEX)] |= CHANNELS_MASK[channelIndex % LAST_BIT_INDEX];
-        });
-
-        data[data.length - 1] &= COUNTER_SIZE;
-
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return data;
-    }
-
-    protected static setValue ( value: number, data: Array<number> ) {
-        let channelValue = value;
-
-        while ( channelValue ) {
-            data.push(EXTENDED_BIT | (channelValue & COUNTER_SIZE));
-            // eslint-disable-next-line no-param-reassign
-            channelValue >>= LAST_BIT_INDEX;
-        }
-
-        const lastValue = data.pop();
-
-        if ( lastValue ) {
-            // just to solve "lastValue is possibly 'undefined'" error
-            data.push(lastValue & COUNTER_SIZE);
-        }
-    }
-
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    protected static setValues ( channels: Map<number, number>, data: Array<number> ): any {
-        channels.forEach(value => {
-            this.setValue(value, data);
-        });
-    }
-
-    // data - only body (without header)
     static fromBytes ( data: Uint8Array ): GetCurrentMul {
         const parameters: IGetCurrentMulParameters = {channels: []};
 
-        const {channels, position} = this.getChannels(data);
-        const channelMap = this.getChannelMap(channels, data, position);
+        const buffer = new CommandBinaryBuffer(data);
 
-        channels.forEach(channelIndex => {
-            const channel = channelMap.get(channelIndex);
+        const channelArray = buffer.getChannels(false);
 
-            if ( channel ) {
-                parameters.channels.push(channel);
-            }
-        });
+        parameters.channels = channelArray.map(channelIndex => ({
```

Comment:
```suggestion
        parameters.channels = channelArray.map((_, channelIndex) => ({
```
?

Reply:
channelIndex it's value of the item in array, not a index of item, fact - they are same, but important to know it and emphasize it with usage